src/pybind/mgr/dashboard/frontend/src/app/core/navigation/about/about.component.scss

```diff
@@ -1,3 +1,12 @@
+@import "bootstrap";
```

Comment:
Rather than importing bootstrap here (which might override modifications made to Bootstrap styles at global scope and add a lot of code here, see https://github.com/ceph/ceph/pull/34283), I'd put that to 'defaults.scss' and import that file from here, or just import what you need (`scss/mixins/_grid.scss`, ...). 

@tspmelo what's your view on this?